src/vaadin-dialog-draggable-mixin.html

```diff
@@ -27,8 +27,9 @@
           const isResizerContainer = e.target === resizerContainer;
           const isResizerContainerScrollbar = e.offsetX > resizerContainer.clientWidth || e.offsetY > resizerContainer.clientHeight;
           const isContentPart = e.target === this.$.overlay.$.content;
-          if ((isResizerContainer && !isResizerContainerScrollbar) || isContentPart) {
-            e.preventDefault();
+          const isDraggable = e.target.classList.contains('draggable');
+          if ((isResizerContainer && !isResizerContainerScrollbar) || isContentPart || isDraggable) {
+            !isDraggable && e.preventDefault();
```

Comment:
This is needed to not prevent default for the case of, for instance, selection of the word by double clicking.

I couldn't come up with a test case to cover that case.